EphraimDev/way-farer
# bug(signup) User can upload image on registration

#### What does this PR do?
Users can upload profile pictures during sign up

#### Description of Task to be completed?
Have the following endpoint working
POST /api/v1/auth/signup

#### How should this be manually tested?
After cloning the repo, cd into it, set the environment variables and run `npm run start:dev`
* Using postman, test for the above enpoint
* The body of the request should contain at least `firstname` `lastname` `email` `password` and `image` as a file

#### What are the relevant pivotal tracker stories?
#167178533
